src/main/java/nom/tam/fits/compression/algorithm/hcompress/HCompressorQuantizeOption.java

```diff
@@ -32,13 +32,18 @@
  */
 
 import nom.tam.fits.compression.algorithm.quant.QuantizeOption;
+import nom.tam.fits.compression.algorithm.rice.RiceCompressOption;
 
 public class HCompressorQuantizeOption extends QuantizeOption {
 
     public HCompressorQuantizeOption() {
         super(new HCompressorOption());
     }
 
+    public HCompressorQuantizeOption(RiceCompressOption compressOption) {
```

Comment:
Here the argument needs to be `HCompressorOption`, to Rice... (I also guess this was one of the lines not covered in the tests...)

Reply:
Working on this now, thanks.